templates/taxi/driver_detail.html

```diff
@@ -22,5 +22,7 @@ <h4>Cars</h4>
     {% empty %}
       <p>No cars!</p>
     {% endfor %}
+      <a href="{% url "taxi:driver-update" pk=driver.id %}" class="btn btn-secondary">Update License number</a>
+      <a href="{% url "taxi:driver-delete" pk=driver.id %}" class="btn btn-danger">Delete driver</a>
```

Comment:
Consider using single quotes consistently in the `{% url %}` tag to match the style used elsewhere in the template. For example, `{% url 'taxi:driver-update' pk=driver.id %}`.